src/translation/translation_pt_PT.ts

```diff
@@ -193,7 +193,7 @@
     <message>
         <location filename="../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2022 The Jamulus Development Team</source>
-        <translation>Copyright (C) 2005-2022 A Equipa de Desenvolvimento do Jamulus</translation>
+        <translation>Direitos de Autor (C) 2005-2022 A Equipa de Desenvolvimento do Jamulus</translation>
```

Comment:
Just as small comment, this doesn't need to be translated

Reply:
About the Unicode copyright © symbol, I agree. Probably it’s worth changing after the upcoming release.